internal/config/config.go

```diff
@@ -23,42 +24,137 @@ import (
 	"golang.org/x/exp/maps"
 )
 
-// TestsConfig represents a part of the dance configuration for tests.
-// (i.e. ferretdb/mongodb/common tests).
+const (
+	// RunnerTypeCommand indicates a command-line test runner.
+	RunnerTypeCommand RunnerType = "command"
+
+	// RunnerTypeGoTest indicates a Go test runner.
+	RunnerTypeGoTest RunnerType = "gotest"
+
+	// RunnerTypeJSTest indicates a JavaScript test runner.
+	RunnerTypeJSTest RunnerType = "jstest"
+
+	// RunnerTypeYCSB indicates a YCSB test runner.
+	RunnerTypeYCSB RunnerType = "ycsb"
+)
+
+// RunnerType represents the type of test runner used in the configuration.
+type RunnerType string
+
+// Stats represent the expected/actual amount of
+// failed, skipped and passed tests.
```

Comment:
FWIW, I think we should follow the same order of fields as in this comment: failed, skipped, passed. Think of it like a traffic lights: 🔴, 🟡, 🟢.